admin_manual/configuration/files/external_storage/google.rst

```diff
@@ -14,97 +14,135 @@ If you already have a Google account, such as Groups, Drive, or Mail, you can
-box. Make sure that your new project is selected, click on **Use Google APIs** , 
-and it takes you to Google's APIs screen. There are many Google APIs; look for 
-the **Google Apps APIs** and click **Drive API.**
+Go to Api overview to select google's API.
 
-.. figure:: images/google-drive3.png
-   :alt: Drive API link.
+.. figure:: images/google_drive/005
+   :alt: API
 
-**Drive API** takes you to the API Manager overview. Click the blue **Enable 
-API** button.
+.. figure:: images/google_drive/006
+   :alt: Google API
 
-.. figure:: images/google-drive4.png
-   :alt: API dashboard.
+.. figure:: images/google_drive/007
+   :alt: Enable
 
-Now you must create your credentials, so click on **Go to credentials**.
-   
-.. figure:: images/google-drive5.png
-   :alt: Create Client ID screen.
+Now you must create your credentials.
    
-For some reason Google warns us again that we need to create credentials. We 
-will use OAuth 2.0.
+.. figure:: images/google_drive/008
+   :alt: Create Credentials
 
-.. figure:: images/google-drive6.png
-   :alt: Another warning to create credentials.
-   
-Now we have to create a consent screen. This is the information in the screen 
-Google shows you when you connect your new Google app to ownCloud the first 
-time. Click **Configure consent screen**. Then fill in the required form fields. 
-Your logo must be hosted, as you cannot upload it, so enter its URL. When 
-you're finished click **Save**.
+First select Web Browser and User data.
+
+.. figure:: images/google_drive/009
+   :alt: Access type and Data
 
-.. figure:: images/google-drive8.png
-   :alt: Creating the consent screen.
+The next screen that opens is **Create OAuth 2.0 Client ID**. Enter your app name. 
 
-The next screen that opens is **Create Client ID**. Check **Web Application**,  
-then enter your app name. **Authorized JavaScript Origins** is your root domain, 
-for example ``https://example.com``, without a trailing slash. You need two 
-**Authorized Redirect URIs**, and they must be in this form::
+.. figure:: images/google_drive/010
+   :alt: Access type and Data
 
+**Authorized JavaScript Origins** is your root domain, 
+for example ``https://example.com``, without a trailing slash. 
+Examples::
+
+  https://example.com
+  http://example.com
+  IP/owncloud  
+
+You need to configure **Authorized Redirect URIs**, and they must be in this form::
+
+  https://example.com/owncloud/index.php/settings/admin?sectionid=storage
   https://example.com/owncloud/index.php/settings/personal?sectionid=storage
-  https://example.com/owncloud/index.php/settings/admin?sectionid=storage 
-  
-Replace ``https://example.com/owncloud/`` with your own ownCloud server URL, 
-then click **Create**.
 
-.. figure:: images/google-drive9.png
-   :alt: Creating client credentials.
+If you configuring storage as an Administrator - choose the admin URI, if you are a user and configuring a storage
```

Comment:
s/If you configuring storage as an Administrator - choose the admin URI/If you are configuring storage as an Administrator - choose the admin URI/g